node/database-paths.md

```diff
@@ -0,0 +1,173 @@
+# Configuring Validator Storage Paths
+
+A validator node is responsible for storing chunk data on disk, and for making that data available when requested.
+Until the V1 protocol is fully deprecated (in favor of the V2 protocol introduced in the `Blazar` release), a validator
+node will store chunk data for both V1 and V2 protocols. The way that data is managed is different between the V1
+protocol and the V2 protocol.
+
+The location on disk where this data is stored is configured by the following two flags:
+
+- `NODE_DB_PATH`: This flag specifies the path where the V1 protocol chunk data is stored. This flag should
+  contain a fully qualified path to a directory where the V1 protocol should store its chunk data.
+- `NODE_LITT_DB_STORAGE_PATHS`: This flag specifies the path where the V2 protocol chunk data is stored.
+  Unlike V1, the V2 data storage engine (LittDB) is capable of spreading data across multiple directories.
+  These directories do not need to be on the same filesystem (e.g. if you want to use multiple disks).
+  To pass in multiple directories, provide a comma-separated list. Each directory should be a fully qualified path.
+
+Until the V1 protocol is fully deprecated, `NODE_DB_PATH` must be set. 
+
+Technically, the new flag `NODE_LITT_DB_STORAGE_PATHS` is optional, since if it is not set then the validator 
+software will store its data in the location specified by `NODE_DB_PATH`. This is not recommended. Eventually,
+the V1 protocol will be disabled, and the `NODE_DB_PATH` flag will be removed along with it. In order to be future
+proof, it is highly recommended to set the `NODE_LITT_DB_STORAGE_PATHS` flag.
+
+# File System Layout
+
+## V1 Protocol
+
+The V1 protocol's disk footprint looks something like this:
+
+```
+${NODE_DB_PATH}
+├── chunk
+│   ├── 000001.log
+│   ├── CURRENT
+│   ├── LOCK
+│   ├── LOG
+│   └── MANIFEST-000000
+```
+
+The `chunk` directory is created by the V1 software inside the directory specified by `NODE_DB_PATH`. Inside
+the `chunk` directory are files maintained by the V1 data storage engine (i.e. `LevelDB`).
+
+## V2 Protocol
+
+The V2 protocol's disk footprint depends on how it is configured.
+
+### Deprecated Configuration: only `NODE_DB_PATH` set
+
+If only `NODE_DB_PATH` is set and `NODE_LITT_DB_STORAGE_PATHS` is not set (not recommended!), then the V2 protocol
+will store its data like this:
+
+```
+${NODE_DB_PATH}
```

Comment:
As you mentioned I can see directories like below,

```
${NODE_DB_PATH}
.
├── chunk
└── chunk_v2_litt
    └── chunks
        ├── keymap
        │   └── data
        └── segments
```

Do you think it would help clarify things if you point out the `chunk` directory, which appears to be used for V1?